tests/conftest.py

```diff
@@ -52,6 +52,21 @@
     from tests.types import ProjectFactory
 
 
+# Enable handling long path names (260) in windows
+# https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#maximum-path-length-limitation
+if sys.platform.startswith("win32"):
```

Comment:
We should probably put this into a fixture with `autouse=True`

Reply:
:heavy_check_mark: